Woosmap/.github
# Modified issue templates and added EPIC template.

# Description
## Issue



## What

With the change of organisation we would like to change our issue templates
## How

Changed to use a Form template, reduced the number of fields and added a extra templates for EPICs and SPIKEs
## Related PRs


# Testing
## Automated tests
n/a


## Steps to test or reproduce

Make a personal repo, copy the files in with the same path `.github/ISSUE_TEMPLATE/epic.yaml` then go to make an issue on that repo and you should see the template to test.


# Deployment
## Strategy



## Migrations

No.
